### DISCUSSION TITLE
[Feature Request] Add the icon sidebar to the "customize toolbar" page and allow for adding icons there.

### DISCUSSION THREAD

**Discussion by YousufSSyed** [Floorp-Projects/Floorp#403] [Feature Request]
### Is your feature request related to a problem?

No.

### Describe the solution you'd like

I'd like to be able to have icons wherever I want on the top or side toolbars. Floorp should add the ability to add icons to the sidebar in the "customize toolbar" page: `hamburger menu > more tools > customize toolbar`. Users could add any icons on whicever toolbar they wish, and can add them to either from the top or bottom of the sidebar.

**Comment by menndouyukkuri** [comment]
The part about vertical tabs was resolved in 11.5.0.
